docs/adding_controllers_without_related_model.md

```diff
@@ -6,16 +6,15 @@ Sometimes you may want to add a custom controller that has no resource
 related to it (for example for a statistics page).
 
 To do that, you must define an `index` route, as only controllers with index
-routes are displayed in the sidebar and then add a custom dashboard:
+routes are displayed in the sidebar, and then add a custom dashboard:
 
 ```erb
-# app/views/admin/stats/index.html.erb 
+# app/views/admin/stats/index.html.erb
 
-<div style="padding: 20px">
+<div>
```

Comment:
The example is copied straight from `spec/example_app/app/views/admin/stats/index.html.erb`. I have removed the padding because it's not relevant here.